### PR TITLE
[TEST] - fix unit-tests

### PR DESCRIPTION
**Pull request by aliakseihaidukou**
now every project has it's own repo path

@dkincade this and https://github.com/pentaho/data-access/pull/600 are for 5.3 branch fixing repository and data-access tests

### CODE REVIEW

**Comment by dkincade** [comment]
@mdamour1976 @pentaho-nbaker please review.

File: repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java [hunk 1]
```diff
@@ -115,7 +111,7 @@
 @ContextConfiguration( locations = { "classpath:/repository.spring.xml",
     "classpath:/repository-test-override.spring.xml" } )
 @SuppressWarnings( "nls" )
-public class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
+public abstract class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
```
Comment (mdamour1976) [inline repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java]: @aliakseihaidukou  I'm curious why this class become abstract during a unit test fix commit.

Comment (aliakseihaidukou) [inline repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java]: @mdamour1976 that's to accent that this class isn't a test class, it is a base for tests that use repository.

**Comment by dkincade** [comment]
Closing this pull request as we are not going to include it before the 5.3 true code freeze...

**Pull request closed without merging**